### PR TITLE
PAYARA-4096 Allow generating instance names from the Admin Console

### PR DESCRIPTION
**Pull request by Pandrex247**
Title.

I've had to remove the _required_ label, so the prompt has changed a little from a text box to a full error message. From my testing this hasn't changed the functionality of the page at all, it's merely aesthetic.

### CODE REVIEW

**Comment by MeroRai** [comment]
jenkins test please

File: appserver/admingui/cluster/src/main/java/fish/payara/cluster/admingui/PayaraClusterHandlers.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package fish.payara.cluster.admingui;
```
Comment (jGauravGupta) [inline appserver/admingui/cluster/src/main/java/fish/payara/cluster/admingui/PayaraClusterHandlers.java]: Copyright header?

File: appserver/admingui/cluster/osgi.bundle [hunk 2]
```diff
@@ -38,4 +38,4 @@
 # holder.
 #
```
Comment (MeroRai) [inline appserver/admingui/cluster/osgi.bundle]: Portions Copyright header?

File: appserver/admingui/cluster/src/main/resources/shared/instanceNew.inc [hunk 3]
```diff
@@ -51,6 +51,8 @@
             <sun:button id="newButton" text="$resource{i18n.button.OK}"
```
Comment (MeroRai) [inline appserver/admingui/cluster/src/main/resources/shared/instanceNew.inc]: Portions Copyright header needs updating.

**Comment by MeroRai** [comment]
@Pandrex247 Tested it locally everything works fine. The only feedback I have is Auto Name could benefit form help text.

**Comment by Pandrex247** [comment]
Jenkins test please

**Pull request merged**